easybuild/easyconfigs/s/sympy/sympy-1.12-foss-2023a.eb

```
@@ -0,0 +1,21 @@
+name = 'sympy'
+version = '1.12'
+
+homepage = 'https://sympy.org/'
+description = """SymPy is a Python library for symbolic mathematics. It aims to
+ become a full-featured computer algebra system (CAS) while keeping the code as
+ simple as possible in order to be comprehensible and easily extensible. SymPy
+ is written entirely in Python and does not require any external libraries."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Since this EasyConfig does not require MPI, I think we could move this in the module hierarchy to the `gfbf` toolchain (requires also a rename the file to `sympy-1.12-gfbf-2023a.eb`):

```suggestion
toolchain = {'name': 'gfbf', 'version': '2023a'}
```